db/migrate/20201009142558_add_role_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddRoleToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :role, :string, null: false, default: ''
+    User.includes(:roles).all.each do |user|
```

Comment:
How are we going to handle this migration once Part 2 is completed?

Reply:
Yeah I'm wondering if this shouldn't be in the migration but rather just be a command we run post-deployment.

Reply:
This migration has to run before part 2, part 2 removes the gem.

Reply:
I think the issue is that this migration will then be invalid once part 2 completes, unless you're saying as part of part 2 you will remove this code from the migration which I think would be the best option.

Reply:
I've updated this to have an up and down for rollback capability

Reply:
Why would you be running this migration again after it executed successfully? I don't understand that part

Reply:
If someone was setting up Sara Alert for the first time they would run into this issue which is a big problem, or even when we reset our databases locally.

Reply:
I eat my words - this will actually be okay because the migration to remove the roles table will happen after this one. It does look like the original rolify migration created both the `roles` table and a `user_roles` table so we'll want to make sure both of those are removed in phase 2. But I think this is fine

Reply:
It's not really the best practice to run migrations when setting up the database for development. Prefer a `schema:load` which would avoid this issue. Same with setting up any new deployments.

Reply:
Gotcha. Might want to update our README then as well.